docs/api_reference/python-api-reference.md

```diff
@@ -11,6 +11,7 @@ SPDX-License-Identifier: MPL-2.0
 .. autoclass:: power_grid_model.PowerGridModel
    :show-inheritance:
 .. autofunction:: power_grid_model.initialize_array
+.. autoclass:: power_grid_model.power_grid_meta_data
```

Comment:
this becomes a mess. i'm working on adding this in #781 instead because the default docstring is unreadable